src/style/Header.module.css

```diff
@@ -1,19 +1,58 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;500;600;700;800&display=swap');
 
+.list {
+  height: 500px;
+  overflow-y: scroll;
+}
+
+/* Books.js styling */
+.list::-webkit-scrollbar {
+  display: none;
+}
+
+.loading {
+  height: 500px;
+  display: flex;
+  gap: 0.8rem;
+  align-items: center;
+  justify-content: center;
```

Comment:
- [ ] Please make sure to increase the left margin for the progress bar, It should be near the `UPDATE PROGRESS` button as on the design.

### Your design

![image](https://github.com/Dorcas126/Book-store-project/assets/83803414/bda3f756-51c0-4704-b1a9-7ae69dc2cb3e)

### Zepline design

![image](https://github.com/Dorcas126/Book-store-project/assets/83803414/f3a944fb-fbfc-4d64-86d4-54591817d6ba)